frontend/packages/jobs/src/prompts/generateSchemaMeta/generateSchemaMeta.ts

```diff
@@ -0,0 +1,74 @@
+import type { Callbacks } from '@langchain/core/callbacks/manager'
+import { ChatPromptTemplate } from '@langchain/core/prompts'
+import { ChatOpenAI } from '@langchain/openai'
+import { dbOverrideSchema } from '@liam-hq/db-structure'
+import { toJsonSchema } from '@valibot/to-json-schema'
+import { parse } from 'valibot'
+
+const dbOverrideJsonSchema = toJsonSchema(dbOverrideSchema)
+
+const SCHEMA_META_TEMPLATE = ChatPromptTemplate.fromTemplate(`
+You are Liam, an expert in database schema design and optimization for this project.
+
+## Your Task
```

Comment:
To be honest, I can imagine that comments will be added, but I can't envision what kind of suggestions for `new relationships` or `additional columns` might be made. However, of course, it seems like a good first step!